sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/implementation/MultipartFormDataHelperTest.java

```diff
@@ -0,0 +1,77 @@
+package com.azure.ai.openai.implementation;
+
+import com.azure.ai.openai.models.AudioTranscription;
+import com.azure.core.http.rest.RequestOptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import com.azure.core.util.BinaryData;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonProviders;
+
+public class MultipartFormDataHelperTest {
+    private MultipartFormDataHelper helper;
+
+    @BeforeEach
+    public void setup() {
+        RequestOptions requestOptions = new RequestOptions();
+        helper = new MultipartFormDataHelper(requestOptions);
+    }
+
+    @Test
+    public void testDeserializeAudioTranscription() throws IOException {
```

Comment:
I think this test should probably be moved to under the `models` package, as it doesn't seem to relate to the `MultipartFormDataHelper` class. 

That being said, are we exercising here anything that isn't exercised in the `testGetAudioTranscription*` tests in the `NonAzureOpenAISyncClient` and all the other files?